okhttp/src/test/java/okhttp3/internal/authenticator/JavaNetAuthenticatorTest.kt

```diff
@@ -0,0 +1,74 @@
+package okhttp3.internal.authenticator
```

Comment:
supernit: copyright

Reply:
We should automate this in spotless :)

Would save you a lot of typing on my reviews :)